Jasmine/Levels/ViewControllers/GameLevelList/GameLevelListViewController.swift

```diff
@@ -30,6 +30,7 @@ class GameLevelListViewController: UIViewController {
             levelsCategoriesToggle.selectedSegmentIndex
                 = newValue ? GameLevelListViewController.builtInLevelsIndex
                            : GameLevelListViewController.customLevelsIndex
+            reloadAllData()
```

Comment:
That's an `IBAction`. If i set `isShowingDefaultLevels` programmatically it won't be called.
Changed `reloadAllData` to  `onLevelsCategoriesSwitched` instead for clearer intent in the commit below.